### PR TITLE
Add welcome email

### PR DESCRIPTION
**Pull request by beaugunderson**
Take a quick look @madprime?

### CODE REVIEW

File: open_humans/templates/email/welcome.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  need to sign up for “news and updates” on your account settings. We really
+  don’t email much! We email to tell Members about new studies and activities,
+  and give updates about progress in current studies.{% endif %}</p>
+
+<p>We believe research should go both ways: find studies to join and share data
+  with, and studies that will share data with you. Build a profile that
+  connects data across studies!</p>
+
+<p>You can even choose to make some data sets public. Joining our Public
+  Sharing Data Study turns on the “public sharing” feature. You’ll be able to
```
Comment (madprime) [inline open_humans/templates/email/welcome.html]: We should link Public Data Sharing

File: open_humans/templates/email/welcome.txt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+We believe research should go both ways: find studies to join and share data
+with, and studies that will share data with you. Build a profile that connects
+data across studies!
+
+You can even choose to make some data sets public. Joining our Public Sharing
+Data Study turns on the “public sharing” feature. You’ll be able to turn
+“public” status on or off for each of your data sets.
+
+Get started by visiting our Welcome page[1].
```
Comment (madprime) [inline open_humans/templates/email/welcome.txt]: Hm... limitations of text. I think it's better to do: `Get started by visiting our Welcome page: {{ welcome_page_url }}`

Also, above:
`Joining our Public Data Sharing study turns on the "public sharing" feature: {{ public_data_url }} You'll be able to turn "public" status on or off for each of your data sets.`

File: open_humans/templates/email/welcome.txt [hunk 3]
```diff
@@ -0,0 +1,34 @@
+Thank you for joining the Open Humans community!
+
+To make sure you get future emails from us, add our email to your “contacts”.
+
+{% if newsletter %}You’ve signed up for news and updates, but without this
+step our emails might be “filtered” by your email client.{% else %}
```
Comment (madprime) [inline open_humans/templates/email/welcome.txt]: I think I'm seeing non-ASCII characters (left and right quotation marks). Would this be an issue, or is non-ASCII fine? (Note, non-ASCII quote marks are also in the HTML email.)

Comment (beaugunderson) [inline open_humans/templates/email/welcome.txt]: I considered replacing them with the standard equivalents but they did come through in the email (and look nicer). We can deploy to staging and you can sign up for a new account to get the welcome email and verify that they work for you too?

**Pull request merged**